### ISSUE TITLE
Standard window icons (Close, Minimize,Maximize) disappear after minimizing when ExtendsContentIntoTitleBar is used

### ISSUE THREAD

**Issue by andreaslennartz** [microsoft/microsoft-ui-xaml#6789]
### Describe the bug

When we use the `ExtendsContentIntoTitleBar` feature (as described in the [docs](https://docs.microsoft.com/en-us/windows/apps/develop/title-bar?tabs=winui3)) with the current WinUI 3 version, the standard icons at the top right (Close, Minimize, Maximize) disappear when the window was minimized. They reappear when hovering over the icons with the mouse.

Before minimizing: 
![image](https://user-images.githubusercontent.com/70487107/156742099-21ac703e-ca18-40ce-a727-a3fc435a3462.png)

After minimizing: 
![image](https://user-images.githubusercontent.com/70487107/156742011-82946be5-55ed-4c76-9c8c-b99cd19c2066.png)

Hovering over the icons:
![image](https://user-images.githubusercontent.com/70487107/156742038-2dfb25cb-814a-454d-ae48-2b33f5eecd44.png)

### Steps to reproduce the bug

Create a new WInUI 3 desktop application (using Visual Studio default tempalte), and add a customized title bar to the Main window following the docs:  https://docs.microsoft.com/en-us/windows/apps/develop/title-bar?tabs=winui3

XAML:
```xml
 <Grid x:Name="AppTitleBar"
            Margin="8,0,120,0">            
                <TextBlock VerticalAlignment="Center" Text="TitleText" />
        </Grid>
```

Code behind:
```C#
 public MainWindow() {
            this.InitializeComponent();
            this.Title = "TitleText";
            //Full customization of title bar: https://docs.microsoft.com/en-us/windows/apps/develop/title-bar?tabs=winui3
            ExtendsContentIntoTitleBar = true;
            SetTitleBar(AppTitleBar);
        }
```

Start the application. Verify that the icons at the top right (Close, Maximize, Minimize) are visible. Minimize the window and bring it back into view again. See that the icons disappeared. 

### Expected behavior

The standard icon (Close, Minimize, Maximize) should *always* be visible. They should never disappear. 

### NuGet package version

WinUI 3 - Windows App SDK 1.0

### Windows app type

- [ ] UWP
- [X] Win32

### Device form factor

Desktop

### Windows version

Windows 11 (21H2): Build 22000, Windows 10 (21H2): Build 19044

**Comment by StephenLPeters** [comment]
@codendone and @pratikone FYI

**Comment by pratikone** [comment]
see if it is fixed in latest 1.0.1 release. I rememebering encountering something similar and providing a fix for it.

**Comment by andreaslennartz** [comment]
@pratikone : yes, I can confirm that this is fixed with 1.0.1 now. 
Thanks for your support!
